nixos/modules/system/etc/setup-etc.pl

```diff
@@ -55,28 +61,31 @@ sub isStatic {
 # in the current one.  For efficiency, don't look under /etc/nixos
 # (where all the NixOS sources live).
 sub cleanup {
-    if ($File::Find::name eq "/etc/nixos") {
+    my $filename = $_;
```

Comment:
```suggestion
    my $filename = shift;
```

Using `shift` is more common to shift elements of the argument list.

Reply:
Suggestion incorrect